apsuite/commisslib/meas_fofb_sysid.py

```diff
@@ -465,10 +484,10 @@ def get_data(self):
         data['prbs_fofbacc_lvl0'] = famsysid.prbs_fofbacc_lvl0
         data['prbs_fofbacc_lvl1'] = famsysid.prbs_fofbacc_lvl1
         data['prbs_bpmpos_enbl'] = famsysid.prbs_bpmpos_enbl
-        data['prbs_bpmposx_lvl0'] = famsysid.prbs_bpmposx_lvl0
-        data['prbs_bpmposx_lvl1'] = famsysid.prbs_bpmposx_lvl1
-        data['prbs_bpmposy_lvl0'] = famsysid.prbs_bpmposy_lvl0
-        data['prbs_bpmposy_lvl1'] = famsysid.prbs_bpmposy_lvl1
+        data['prbs_bpmposx_lvl0'] = _np.roll(famsysid.prbs_bpmposx_lvl0, -1)
+        data['prbs_bpmposx_lvl1'] = _np.roll(famsysid.prbs_bpmposx_lvl1, -1)
+        data['prbs_bpmposy_lvl0'] = _np.roll(famsysid.prbs_bpmposy_lvl0, -1)
+        data['prbs_bpmposy_lvl1'] = _np.roll(famsysid.prbs_bpmposy_lvl1, -1)
```

Comment:
I think it would be best to change the name of these dict members, so we don't accidentally use them from old files thinking they are ordered when they aren't.